content/releases/Release Notes 07.30.25.md

```
@@ -0,0 +1,30 @@
+
+FIXED
+
+- Summarizer module to ignore empty summaries and pass appropriate one to get_context
+- Structured Outputs calls with OpenAI provider to pass strict=True to Pydantic Schema
+
+## Links
+
+- [Sign-up for Honcho](https://app.honcho.dev/) & start building personalized agent experiences
+- [Join our Discord](https://discord.gg/honcho) & tell us what you're working on
+- [Visit our open-source repo](https://github.com/plastic-labs/honcho) & get your hands dirty
+- [Check out the docs](https://docs.honcho.dev)
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character

(MD047, single-trailing-newline)